lib/passwordValidation.js

```diff
@@ -0,0 +1,9 @@
+// check password validation
```

Comment:
Nit: File names should be in [kebab case](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#kebab-case).

[What is nit?](https://josipmisko.com/posts/code-review-nit)